source/parser/x86_64/classifiers.hpp

```diff
@@ -241,7 +241,13 @@ namespace smeagle::x86_64 {
     return {RegisterClass::INTEGER, RegisterClass::NO_CLASS, "Enum"};
   }
 
-  inline classification classify(st::typeFunction *t) { return {}; }
+  inline classification classify(st::typeFunction *t) {
+    auto [underlying_type, ptr_cnt] = unwrap_underlying_type(t);
+    if (ptr_cnt > 0) {
+      return classify_pointer(ptr_cnt);
+    }
+    return {};
```

Comment:
We should probably throw an exception here, but we can punt on that since general error handling hasn't been established for the other classifiers yet (that's something we should think about at some point).

Reply:
Agree, I'll open an issue.